src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -41,11 +47,34 @@ public double getVel() {
 
     public void stop() {
         turret.stopMotor();
+    }   
+
+    public void setPos(int position){
```

Comment:
This resets the sensor to some value. I don't think we're going to need this? Or maybe just a "reset" method